src/test/app/Flow_test.cpp

```diff
@@ -1392,9 +1396,16 @@ struct Flow_test : public beast::unit_test::suite
         auto const bob = Account("bob");
         auto const gw = Account("gw");
         auto const USD = gw["USD"];
+        auto const usdC = USD.currency;
 
         env.fund(XRP(10000), alice, bob, gw);
```

Comment:
I don't want to mess with `Env::fund` in this PR. I'd guess we almost always want a `close` after a `fund`, so adding it to `fund` is probably a good idea.